framework/src/media/Kconfig

```diff
@@ -9,6 +9,7 @@ config MEDIA
 	depends on AUDIO
 	select HAVE_CXX
 	select HAVE_CXXINITIALIZE if BUILD_FLAT
+	select MEDIA_RESAMPLER
```

Comment:
I don't mean this, it should be removed here.

Reply:
I think this required.